docs/user_guide/architecture.md

```diff
@@ -536,11 +536,13 @@ all inference requests in a sequence are routed to the same model
 instance and then uses the [dynamic
 batcher](model_configuration.md#dynamic-batcher) to batch together
 multiple inferences from different sequences into a batch that
-inferences together.  With this strategy the model must typically use
-the CONTROL_SEQUENCE_CORRID control so that it knows which sequence
-each inference request in the batch belongs to. The
-CONTROL_SEQUENCE_READY control is typically not needed because all
-inferences in the batch will always be ready for inference.
+inferences together. With this strategy the model may need to use the
+CONTROL_SEQUENCE_CORRID control so that it knows which sequence each inference
+request in the batch belongs to. This is typically not required if the model is
+using [implicit state management](#implicit-state-management), since the
+sequence batcher will manage the state associated with each sequence. The
+CONTROL_SEQUENCE_READY control is typically not needed because all inferences
+in the batch will always be ready for inference.
```

Comment:
I think "will always be ready for inference" can be elaborated a bit more, the wording doesn't have to be exact but something like "..., comparing to the case of direct sequence batcher where a batch slot has to reserved for the same sequence which may not be ready in the batch if the request hasn't arrived"